src/libs/error.test.ts

```diff
@@ -1,114 +1,36 @@
 import { sessionTimedOutErrorMessage } from 'src/auth/auth-errors';
-import { reportError as reportErrorFallback, withErrorReporter } from 'src/libs/error';
+import { reportError as reportErrorFallback } from 'src/libs/error';
 import { notify } from 'src/libs/notifications';
 import { mockNotifications } from 'src/testing/test-utils';
 
 jest.mock('src/libs/notifications');
 
-describe('report', () => {
+describe('reportError - using terra notificationsProvider fallback', () => {
```

Comment:
More hints that the name can be improved... here it's called a "notifications provider".